packages/graphql-mocks/src/pack/pack.ts

```diff
@@ -9,7 +7,7 @@ export const pack: Packer = async function pack(
   middlewares = [],
   packOptions = defaultPackOptions,
 ) {
-  middlewares = [...middlewares, embed({ wrappers: [embedPackOptionsWrapper] })];
```

Comment:
This causes a huge slowdown because it was wrapping all resolvers for every field on a schema. In large schemas this ends up being very costly.